lib/fastlane/plugin/wpmreleasetoolkit/helper/github_helper.rb

```diff
@@ -7,8 +7,19 @@ module Fastlane
 
   module Helper
     class GithubHelper
+      def self.github_token
```

Comment:
Ruby conventions suggest to end the name with a `!` to indicate the method is throwing
```suggestion
      def self.github_token!
```